internal/freelist/freelist.go

```diff
@@ -0,0 +1,86 @@
+package freelist
+
+import (
+	"sort"
+
+	"go.etcd.io/bbolt/internal/common"
+)
+
+type Freelist interface {
```

Comment:
Refer to https://github.com/golang/go/blob/ea537cca314d9da5365eeefcc375410c76e93b36/src/sort/sort.go#L14
```suggestion
type Interface interface {
```

Reply:
done